urlchecker/core/check.py

```diff
@@ -45,7 +45,7 @@ def __init__(
         # Save run parameters
         self.white_listed_files = white_listed_files or []
         self.include_patterns = include_patterns or []
-        self.print_all = print_all
+        self.print_level = print_level
```

Comment:
For all the places that print_level is set from a client, we have to assume that a user might set it to None, thinking this is default or "unset." So instead of just blindly setting it like this, let's do something like:

```bsah
self.print_level = print_level or "none"
```
That way if it's set to an empty string or None, it will not break the client.

Reply:
But since we favor verbosity, the default should be:

```python
self.print_level = print_level or "all"
```

Reply:
makes sense. I will make the change.